metadata/src/main/java/org/apache/kafka/controller/ClusterControlManager.java

```diff
@@ -485,12 +541,36 @@ Iterator<UsableBroker> usableBrokers() {
             id -> brokerRegistrations.get(id).rack());
     }
 
+    /**
+     * Returns true if the broker is in fenced state; Returns false if it is
+     * not or if it does not exist.
+     */
     public boolean unfenced(int brokerId) {
```

Comment:
Minor but I am pretty sure that we don't use this method anymore in `src/main`. If so, I say that we just remove it.

Reply:
That's right. However, it is used in many places in the tests. I haven't found a good way to replace it in tests that is as convenient as this predicate. I would keep it.